README.md

```diff
@@ -68,11 +68,10 @@ Each var that is recorded can be customized with options:
                   fn; if it returns falsy, the call will be passed to
                   through to the original function both during recording
                   and playback.
-
-## TODO
-
-* Add a better way to re-record than deleting cassette files.
-  Maybe an environment variable?
```

Comment:
The TODO should stay here, as there are still use cases that this feature doesn't address.